lang/tags/operators.py

```diff
@@ -0,0 +1,204 @@
+mod.tag(
+    "code_operators_lambda", desc="Tag for enabling commands for anonymous functions"
+)
+mod.tag("code_operators_math", desc="Tag for enabling mathematical operator commands")
+mod.tag("code_operators_pointer", desc="Tag for enabling pointer operator commands")
+
+mod.list("code_operators_array", desc="List of code operators for arrays")
+mod.list("code_operators_assignment", desc="List of code operators for assignments")
+mod.list("code_operators_bitwise", desc="List of code operators for bitwise operations")
+mod.list("code_operators_lambda", desc="List of code operators for anonymous functions")
+mod.list(
+    "code_operators_math", desc="List of code operators for mathematical operations"
+)
+mod.list("code_operators_pointer", desc="List of code operators for pointers")
+
+
+Operator = str | Callable[[], None]
+
+
+class Operators(TypedDict):
+    # code_operators_array
+    SUBSCRIPT: NotRequired[Operator]
+
+    # code_operators_assignment
+    ASSIGNMENT: NotRequired[Operator]
+    ASSIGNMENT_OR: NotRequired[Operator]
+    ASSIGNMENT_SUBTRACTION: NotRequired[Operator]
+    ASSIGNMENT_ADDITION: NotRequired[Operator]
+    ASSIGNMENT_MULTIPLICATION: NotRequired[Operator]
+    ASSIGNMENT_DIVISION: NotRequired[Operator]
+    ASSIGNMENT_MODULO: NotRequired[Operator]
+    ASSIGNMENT_INCREMENT: NotRequired[Operator]
+    ASSIGNMENT_BITWISE_AND: NotRequired[Operator]
+    ASSIGNMENT_BITWISE_OR: NotRequired[Operator]
+    ASSIGNMENT_BITWISE_EXCLUSIVE_OR: NotRequired[Operator]
+    ASSIGNMENT_BITWISE_LEFT_SHIFT: NotRequired[Operator]
+    ASSIGNMENT_BITWISE_RIGHT_SHIFT: NotRequired[Operator]
+
+    # code_operators_bitwise
+    BITWISE_AND: NotRequired[Operator]
+    BITWISE_OR: NotRequired[Operator]
+    BITWISE_NOT: NotRequired[Operator]
+    BITWISE_EXCLUSIVE_OR: NotRequired[Operator]
+    BITWISE_LEFT_SHIFT: NotRequired[Operator]
+    BITWISE_RIGHT_SHIFT: NotRequired[Operator]
+
+    # code_operators_lambda
+    LAMBDA: NotRequired[Operator]
+
+    # code_operators_math
+    MATH_SUBTRACT: NotRequired[Operator]
+    MATH_ADD: NotRequired[Operator]
+    MATH_MULTIPLY: NotRequired[Operator]
+    MATH_DIVIDE: NotRequired[Operator]
+    MATH_MODULO: NotRequired[Operator]
+    MATH_EXPONENT: NotRequired[Operator]
+    MATH_EQUAL: NotRequired[Operator]
+    MATH_NOT_EQUAL: NotRequired[Operator]
+    MATH_GREATER_THAN: NotRequired[Operator]
+    MATH_GREATER_THAN_OR_EQUAL: NotRequired[Operator]
+    MATH_LESS_THAN: NotRequired[Operator]
+    MATH_LESS_THAN_OR_EQUAL: NotRequired[Operator]
+    MATH_AND: NotRequired[Operator]
+    MATH_OR: NotRequired[Operator]
+    MATH_NOT: NotRequired[Operator]
+    MATH_IN: NotRequired[Operator]
+    MATH_NOT_IN: NotRequired[Operator]
+
+    # code_operators_pointer
+    POINTER_INDIRECTION: NotRequired[Operator]
+    POINTER_ADDRESS_OF: NotRequired[Operator]
+    POINTER_STRUCTURE_DEREFERENCE: NotRequired[Operator]
+
+
+@mod.action_class
+class Actions:
+    def code_operator(id: str):
```

Comment:
`id` is a Python builtin function so would recommend against using it as a parameter. (Surprised nothing warned about this!)

Reply:
Oh that ship has thoroughly sailed already :D
https://search.talonvoice.com/search/?q=file%3A%5C.py%24%20%5C(id%3A%20&fold_case=auto&regex=true&context=true